### PR TITLE
Clarify behaviour of converters for primitive wrappers.

### PR DESCRIPTION
**Pull request by radcortez**
I'm adding a clarification that Converters targeting wrapper primitive types should also be applied to the corresponding primitive types.

Not sure how each implementation is handling this, but I think it makes sense.

Also, it seems there is no TCK for this, so if we agree with this I'll add some.

### CODE REVIEW

File: spec/src/main/asciidoc/converters.asciidoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ All built-in ``Converter``s have the `@Priority` of `1`.
 === Adding custom Converters
 
 A custom `Converter` must implement the generic interface `org.eclipse.microprofile.config.spi.Converter`.
-The Type parameter of the interface is the target type the String is converted to.
+The Type parameter of the interface is the target type the String is converted to. If your converter targets a wrapper of a primitive type (e.g. `java.lang.Integer`), the converter applys to both the wrapper type and the primitive type (e.g `int`)
```
Comment (dmlloyd) [inline spec/src/main/asciidoc/converters.asciidoc]:
```suggestion
The Type parameter of the interface is the target type the String is converted to. If your converter targets a wrapper of a primitive type (e.g. `java.lang.Integer`), the converter applies to both the wrapper type and the primitive type (e.g `int`)
```

Just a minor spelling fix.

Comment (radcortez) [reply spec/src/main/asciidoc/converters.asciidoc]: Ops. Thank you!

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by radcortez** [comment]
I did on separate PR:
https://github.com/eclipse/microprofile-config/pull/519

**Comment by Emily-Jiang** [comment]
Cool. Thanks @radcortez ! Will you be able to add char and Character combination?

**Comment by radcortez** [comment]
Sure.

**Comment by Emily-Jiang** [comment]
Will you be able to open an issue for the two PRs so that we can track what issues we are delivering for which mileston? Thanks

**Comment by radcortez** [comment]
Sure.

**Pull request merged**